config/initializers/app_keys.rb

```diff
@@ -1,7 +1,9 @@
-ENV["email_user_name"] = 'answerawesome'
-ENV["email_password"]  = 'Sup3r$ecret$$'
+ENV['email_user_name'] = 'answerawesome'
+ENV['email_password']  = 'Sup3r$ecret$$'
 ENV['TWITTER_CONSUMER_KEY'] = 'ftCIdGCfv1w9cUeo1MNLYc2xd'
-ENV['TWITTER_CONSUMER_SECRET'] = 'oEO3cHGR9noj5j5sgNBNIW8YU1ufuiU5trfRt8HgGcJF0Y4y3P'
+ENV[
+  'TWITTER_CONSUMER_SECRET'
+  ] = 'oEO3cHGR9noj5j5sgNBNIW8YU1ufuiU5trfRt8HgGcJF0Y4y3P'
 
 ENV['FACEBOOK_APP_ID'] = '2244085749149962'
 ENV['FACEBOOK_APP_SECRET'] = '1ce1e1f0639eff17086811b44e2dbbc2'
```

Comment:
Something is really wrong here bc this file should not be going up to github.
we need to make sure to put it on gitignore.